.nvmrc

```diff
@@ -0,0 +1 @@
+v22.13.0
```

Comment:
I added this file as a proposal.

I've seen there is no information about which node version should you use when running this app. I think specifying the one recommended node version to run this code could be beneficial and provide a sort of the packages stability. It seems to me that the best node version could be the one that is used on the prod.